openvdb_houdini/houdini/SOP_OpenVDB_Sample_Points.cc

```diff
@@ -424,24 +556,35 @@ VDB_NODE_OR_CACHE(VDB_COMPILABLE_SOP, SOP_OpenVDB_Sample_Points)::cookVDBSop(OP_
                         grid, threaded, aGdp, attribHandle, &scalarInterrupt);
                     theSampler.sample();
 
+                    // loop over vdb point grids
+                    sampleToVdbPointsGrids<cvdb::FloatGrid>(vdb, attributeName,
```

Comment:
It might be worth supporting threaded and interrupt parameters to be consistent with the Houdini point sampling? (As an aside, much of the existing point functionality is currently missing the threaded option, which I think is probably a worthwhile addition).